torch/tensor.py

```diff
@@ -41,14 +41,27 @@ def __reduce_ex__(self, proto):
         # See Note [Don't serialize hooks]
         torch.utils.hooks.warn_if_has_hooks(self)
         if self.is_quantized:
+            if self.qscheme() == torch.per_tensor_affine:
```

Comment:
We still support torch.per_tensor_symmetric and torch.per_channel_symmetric as valid qschemes. They are used only by the observer currently. We should add checks for these too whenever we have a qscheme based check.

Reply:
this code will never be hit by observers - you can't create a tensor with that quantizer